### DISCUSSION TITLE
When can CentOS 8 be supported

### DISCUSSION THREAD

**Discussion by xw-l** [fail2ban/fail2ban#3105] [General]
as the title

**Comment by sebres** [comment]
The question is a bit unclear... (@fail2ban/maintainers)
If I'm not wrong, fail2ban will be already provided for CentOS as package - see https://pkgs.org/search/?q=fail2ban.

Theoretically, if you're familiar, fail2ban can be also installed from source (see https://github.com/fail2ban/fail2ban/wiki/How-to-install-or-upgrade-fail2ban-manually#manual-installation-from-source), but you have to make a bit more configuration probably as for paths and CentOS related settings (quasi repeat the work the maintainer of your distribution already made for you).

If you mean to release it here on GH, then possibly never - it is a matter of maintainers and we have not enough time to make a release for every \*nix distribution of the world. The upstream packages for Debian are available for several reasons: a) historical and because ourselves using it; b) it is also base system of CI's

**Comment by xw-l** [comment]
thx my bro ,you are nice guy.have a nice day,and wlecome you come to china.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "fail2ban/fail2ban"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年9月22日(星期三) 晚上6:02
***@***.***&gt;;
抄送:&nbsp;" ***@***.******@***.***&gt;;
主题:&nbsp;Re: [fail2ban/fail2ban] When can CentOS 8 be supported (Discussion #3105)





 
The question is a bit unclear... ***@***.***/maintainers)
 If I'm not wrong, fail2ban will be already provided for CentOS as package - see https://pkgs.org/search/?q=fail2ban.
 
Theoretically, if you're familiar, fail2ban can be also installed from source (see https://github.com/fail2ban/fail2ban/wiki/How-to-install-or-upgrade-fail2ban-manually#manual-installation-from-source), but you have to make a bit more configuration probably as for paths and CentOS related settings (quasi repeat the work the maintainer of your distribution already made for you).
 
If you mean to release it here on GH, then possibly never - it is a matter of maintainers and we have not enough time to make a release for every *nix distribution of the world. The upstream packages for Debian are available for several reasons: a) historical and because ourselves using it; b) it is also base system of CI's
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by hobbes1069** [comment]
It is provided by the Fedora EPEL Repositories:

https://docs.fedoraproject.org/en-US/epel/

**Comment by xw-l** [comment]
does‘t matter，English is too difficult，anyway，Thank you again，for your answer。




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "fail2ban/fail2ban"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年9月22日(星期三) 晚上7:57
***@***.***&gt;;
抄送:&nbsp;" ***@***.******@***.***&gt;;
主题:&nbsp;Re: [fail2ban/fail2ban] When can CentOS 8 be supported (Discussion #3105)





 
It is provided by the Fedora EPEL Repositories:
 
https://docs.fedoraproject.org/en-US/epel/
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
